data/index.js

```diff
@@ -119,5 +119,12 @@ export const query = ( mapSelectorsToProps ) => ( WrappedComponent ) => {
  * @return {*} The selector's returned value.
  */
 export const select = ( reducerKey, selectorName, ...args ) => {
+	const selector = get( selectors, [ reducerKey, selectorName ] );
+	if ( typeof selector !== 'function' ) {
+		// eslint-disable-next-line no-console
+		console.error( `Invalid selector called, with name "${ selectorName }" for reducer "${ reducerKey }".` +
+			' Make sure the reducerName and selectorName are correct!' );
+		return undefined;
+	}
```

Comment:
Should we add the same check line 103 or maybe reuse the same function?

Reply:
Fixed.